winml/adapter/winml_adapter_model.cpp

```diff
@@ -228,54 +228,54 @@ ORT_API_STATUS_IMPL(winmla::CloneModel, const OrtModel* in, OrtModel** out) {
   API_IMPL_END
 }
 
-ORT_API_STATUS_IMPL(winmla::ModelGetAuthor, const OrtModel* model, const char** const author, size_t* len) {
+ORT_API_STATUS_IMPL(winmla::ModelGetAuthor, _In_ const OrtModel* model, _Out_ const char** const author, _Out_ size_t* len) {
   API_IMPL_BEGIN
```

Comment:
why is "_Out_ const char** const author" not _Outptr_?

Reply:
several other places too. is there something special about char?

Reply:
Hmmm, I was just following the existing paradigm that the implementor of this code used for the interface definition. My guess is that they are thinking of the **char more like a pointer to a string. _Out_ and _Out_ ptr in effect enforce the exact some conditions so its more of a style thing I think.